io.github.FailurePoint.RandomNumberFive.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "io.github.FailurePoint.RandomNumberFive",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "random-number-five",
+    "finish-args" : [
+        "--share=network",
```

Comment:
why does a random number generator need the network?

Reply:
it was inserted by GNOME Builder by default, I thought I removed it but I had to rebuild the software a few so I must have missed it on this one.